### PR TITLE
fix(TBD-5163): Remove the dependencies signature in the fat jar

### PR DESCRIPTION
**Pull request by rdubois**

### CODE REVIEW

File: test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java [hunk 1]
```diff
@@ -165,4 +163,26 @@ public void getAllFilesFromFolder3() throws IOException {
...
+    @Test
+    public void deleteFilesTest() throws IOException {
+
+        folder1.newFile("myfile.txt"); //$NON-NLS-1$
+        folder1.newFile("myfile2.txt"); //$NON-NLS-1$
+        folder1.newFile("myfile2.zip"); //$NON-NLS-1$
+
+        File folder1File = folder1.getRoot();
+
+        FileUtils.deleteFiles(folder1File, name -> false);
```
Comment (sgandon) [inline test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java]: You may use () -> false here I think.

File: test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java [hunk 2]
```diff
@@ -165,4 +163,26 @@ public void getAllFilesFromFolder3() throws IOException {
...
+        assertEquals(folder1File.listFiles().length, 3);
+
+        FileUtils.deleteFiles(folder1File, name -> name.endsWith(".txt")); //$NON-NLS-1$
+        assertEquals(folder1File.listFiles().length, 1);
```
Comment (sgandon) [inline test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java]: I would suggest that you also check this is the right file (myfile2.zip) that is left.

File: test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java (hunk 1 above)
Comment (rdubois) [inline test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java]: It wouldn't compile.

File: test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java (hunk 2 above)
Comment (rdubois) [inline test/plugins/org.talend.utils.test/src/org/talend/utils/files/FileUtilsTest.java]: +1. Done

**Pull request merged**